CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# source{d} Contributing Guidelines
```

Comment:
The contributing guide should explain how to setup the development environment.

Reply:
Shall we add it to the CONTRIBUTING.md template? https://github.com/src-d/guide/blob/master/engineering/documents/CONTRIBUTING.md

Reply:
Maybe we should just add a `Tests` section to README? So that there is: Installation, Usage, Build, Tests.